specificatie/verblijfplaats.yaml

```diff
@@ -127,7 +127,7 @@ components:
         - $ref: '#/components/schemas/AbstractVerblijfplaatsBeperkt'
         - type: object
           description: |
-            * **functieAdres** - wordt gevuld op basis van de waarden die voorkomen in de tabel 'soortadres' uit de Haal-Centraal-BRP-tabellen-bevragen API.
+            * **functieAdres** - wordt gevuld op basis van de waarden die voorkomen in de tabel 'Soort_Adres' uit de Haal-Centraal-BRP-tabellen-bevragen API.
```

Comment:
waarom eigenlijk "soort" van het adres? Het gaat om de functie van het adres. De soort van het adres is opgenomen in "type" (soort en type zijn volgens mij synoniemen).